charts/agent/templates/_config-exporters.tpl

```diff
@@ -2,21 +2,21 @@
 otlphttp/observe/base:
     endpoint: "{{ .Values.observe.collectionEndpoint | toString | trimSuffix "/" }}/v2/otel"
     headers:
-        authorization: "Bearer {{ .Values.observe.token }}"
```

Comment:
Question: it seems that historically our agents does expects the "Bearer" string to be outside of the token [1]. But it seems you are expect that string in the token here? Is that intentional?

(And I feel like the env var name probably should be `TOKEN`, instead of `OBSERVE_TOKEN`, given how you set it up. But I might be wrong. If your local testing works, please ignore me :P

[1]
https://github.com/observeinc/helm-charts/blob/c62a44b5bdf29f2b9da1c63db8e9b31506ead36f/charts/traces/values.yaml#L106
https://github.com/observeinc/manifests/blob/a4261459b61fc4fe2e17684f3d5bfcbb8cd1ea88/bases/logs/m/fluent-bit.conf#L95

Reply:
After talking to @obs-gh-alexlew figured out that agent automatically does substitution  if TOKEN env var exists

Reply:
Also agent does Bearer portion as well